docs/source/community_guides.rst

```diff
@@ -1,33 +0,0 @@
-.. ...........................................................................
```

Comment:
Did we intend to remove this doc file?

Reply:
Confirmed w/ Dipti - these pages are intended to be removed. Cleared it with Bryant P.